### PR TITLE
navigate to community home screen

### LINKED ISSUE #15491: User is not redirected to community page after finishing onboarding
followup of https://github.com/status-im/status-mobile/pull/15370 cc @jo-mut

Expected result: according to the feature [description](https://github.com/status-im/status-mobile/issues/15033#issue-1574817901 ) 
"should direct user to communities home page afterwards"

**Actual result:** user is redirected to Jump to screen

https://user-images.githubusercontent.com/97245802/227504770-dfcd2192-0a36-45c6-bbb4-c43048c97f8c.mp4

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by jo-mut**
fixes #15491

### Summary
This PR directs user to communities home screen after enable notifications.

### CODE REVIEW

File: src/status_im2/contexts/onboarding/enable_notifications/view.cljs [hunk 1]
```diff
@@ -10,6 +10,7 @@
     [status-im2.contexts.onboarding.common.background.view :as background]
     [status-im2.contexts.onboarding.enable-notifications.style :as style]))
 
+
```
Comment (flexsurfer) [inline src/status_im2/contexts/onboarding/enable_notifications/view.cljs:13]: extra line

File: src/status_im2/contexts/shell/view.cljs [hunk 2]
```diff
@@ -137,16 +138,20 @@
   []
   [:f>
    (fn []
-     (let [shared-values (animation/calculate-shared-values)]
+     (let [shared-values (animation/calculate-shared-values)
+           show-shell?   (reagent/atom false)]
```
Comment (Parveshdhull) [inline src/status_im2/contexts/shell/view.cljs]: Hi, we already have functionality to set default selected tab.
Its added to create-only-event, but not sure why that event didn't got called in mentioned issue when creating account.
If it follows some other flow or event, please add this change there too
https://github.com/status-im/status-mobile/blob/ac2d10bc5d2001df8a45be1834dc81d3dcbe2ceb/src/status_im/multiaccounts/login/core.cljs#L522

**Review by Parveshdhull — CHANGES_REQUESTED** [review]

**Review by Parveshdhull — APPROVED** [review]
Thank you for fixing it

**Review by briansztamfater — APPROVED** [review]
LGTM! Please run `make lint-fix` to get rid of linting issues

**Comment by VolodLytvynenko** [comment]
Hi @jo-mut thank you for PR. One issue is found:

## ISSUE 1:  The user is not navigated to the communities home page after 'Sign in' flow is passed
#### Steps to reproduce:
1. User_A: Go to profile page -> syncing ->Tap 'Sync a new device' -> tap 'Setup Syncing' button -> Enter user's password -> Tap 'Generate scan' button
2. User_A: Leave the generated code on the screen
3. User_B: Open 'Status' app
4. User_B: Tap 'Sign in'
5. User_B: Enable camera -> point the camera at the generated QR code
6. Pass next onboarding pages


#### Actual result:
The 'jump to'  page is opened after onboarding is passed

https://user-images.githubusercontent.com/52490791/228780361-71c8921e-a278-4674-8ed7-7966c92e68e2.mp4

#### Expected result:
The user is redirected to the communities home page

**Comment by VolodLytvynenko** [comment]
Hi, @jo-mut I see some new commits were pushed. Let me know if this PR is ready for tests

**Comment by jo-mut** [comment]
@VladimrLitvinenko  I will let you know once i am okay with this

**Comment by jo-mut** [comment]
@VladimrLitvinenko its good now to proceed. I had not resolved this before in the case of scan to login flow but now should be fine in both `scan to login` and `I am new to status`

**Comment by VolodLytvynenko** [comment]
Hi @jo-mut. Thank you for your work. Issue 1 is fixed. No issues from my side. PR is ready to be merged

**Pull request merged**